flank-scripts/src/main/kotlin/flank/scripts/utils/Env.kt

```diff
@@ -1,6 +1,5 @@
 package flank.scripts.utils
 
-fun getEnv(key: String): String =
-    System.getenv(key) ?: throw RuntimeException("Environment variable '$key' not found!")
-
+fun getEnv(key: String, default: String? = null): String =
+    System.getenv(key) ?: default ?: throw RuntimeException("Environment variable '$key' not found!")
```

Comment:
nit: With this function changed we could also update https://github.com/Flank/flank/blob/238e3790d59097a54ca7e97326d0b1a84f381b61/flank-scripts/src/main/kotlin/flank/scripts/utils/Git.kt#L9 and make it a bit simpler